classes/i18n/Locale.php

```diff
@@ -365,54 +365,38 @@ public function getScripts(?string $locale = null): Scripts
     /**
      * Get the formatted locale display names with country if same language code present multiple times
      * 
-     * @param   array $filterByLocales  Optional list of locales code to filter by the returned formatted names list
-     * @param   array $locales          Optional list of availabel all locales
-     * @param   int   $langLocaleStatus The const value of one of LocaleMetadata:LANGUAGE_LOCALE_*
+     * @param   array $filterByLocales          Optional list of locales code to filter by the returned formatted names list
+     * @param   array $locales                  Optional list of availabel all locales
+     * @param   int   $langLocaleStatus         The const value of one of LocaleMetadata:LANGUAGE_LOCALE_*
+     * @param   bool  $omitLocaleCodeInDisplay  Should leave out the locale code from display. By deault leave out.
      * 
-     * @return  array                   The list of locales with formatted display name
+     * @return  array                           The list of locales with formatted display name
      */
-    public function getFormattedDisplayNames(array $filterByLocales = null, array $locales = null, int $langLocaleStatus = LocaleMetadata::LANGUAGE_LOCALE_WITH): array
+    public function getFormattedDisplayNames(array $filterByLocales = null, array $locales = null, int $langLocaleStatus = LocaleMetadata::LANGUAGE_LOCALE_WITH, bool $omitLocaleCodeInDisplay = true): array
```

Comment:
What do you think about homogenizing the locale code's display or non-display into the `$langLocaleStatus` option? It seems like that option could handle the full set of display cases.

Reply:
@asmecher I thought about it but could not come up with a good solution . Right now we have 3 cases defined as `LANGUAGE_LOCALE_*` and each define different presentation. but adding `locale code` to each of it multiply it and make it harder to properly identify each presentation case or so it seems to me .